### DISCUSSION TITLE
Replace: Some of the channels in the China playlist are not available

### DISCUSSION THREAD

**Discussion by Lemonawa** [iptv-org/community#193] [Show and tell]
### I have verified that

- [X] I use the GitHub playlist link and not a downloaded/cached version of the file
- [X] My TV player has updated this playlist after 0:00 GMT and is set to update it daily
- [X] I am aware that the channel source may have temporal outages. It is broken/absent from the playlist for couple of hours or days

### Channel Name

https://gist.github.com/Lemonawa/4b25d33bdda5ea632ee64307fda6b36c

### What happened to the stream?

It is now offline (disappeared from the playlist)

### Playlist

https://iptv-org.github.io/iptv/countries/cn.m3u

**Comment by BellezaEmporium** [comment]
Thank you for this list. I'm making a PR reflecting some changes.
